### ISSUE TITLE
Dolphin fails to display files moved by trm, as it is missing a .trashinfo file

### ISSUE THREAD

**Issue by ghost** [DiegoMagdaleno/trm#3]

**Comment by ghost** [comment]
the info it wants is:
The path of where the file was before it was deleted
The Deletion Date in some weird format (2020-08-11T15:00:03)

**Comment by ghost** [comment]
the files in .local/share/Trash/info are named like this
filename.trashinfo where the filename is the full file or foldername with its extension

**Comment by DiegoMagdaleno** [comment]
Thank you for reporting this.

I will look more into some other popular linux file managers.

And fix this by writting a lib that defines actions dependant on the file desktop enviroment.

**Comment by ghost** [comment]
it appears like nautilus/nemo dont care about this

**Comment by ghost** [comment]
im testing other file managers right now

**Comment by DiegoMagdaleno** [comment]
#

**Comment by ghost** [comment]
yeah literally every other gui file manager does not care that I found on 
https://wiki.archlinux.org/index.php/Category:File_managers

**Comment by DiegoMagdaleno** [comment]
Would you mind sending me the output of what you do if you run the command `tree` in the location where the .trashinfo files are located?

Thanks.

Also,if you dont mind make sure it is on:

```-this
     | - Dir
     |     |--- Another dir 
     | -Format
```

**Comment by ghost** [comment]
❯ tree
.
├── directorysizes
├── files
│   ├── output.txt
│   └── test1
└── info
    └── test1.trashinfo

2 directories, 4 files

output.txt is a file I deleted with trm and test1.trashinfo is a file I made myself seeing if you could just make your own trashinfo file

also it looks like if you make a trashinfo file yourself dolphin doesnt see the file in the trash still until you restart it, probably because it doesnt check for new trashinfo files since it assumes only it makes them

**Comment by ghost** [comment]
I'm also not sure what "directorysizes" is for but it doesnt seem to matter and nothing was in it
